bin/docker-dev-shell

```diff
@@ -71,6 +72,11 @@ if [ -n "$RUNNING" ]; then
   fi
 fi
 
+if [ -z "$PROJECT_LOCATION" ]; then
+  # a helper for window subsystem for linux
+  PROJECT_LOCATION=$(pwd)
```

Comment:
I'm confused why we need to set `PROJECT_LOCATION=$(pwd)` again here, when it's already set on line 5? Why can't we directly use `$(pwd)` on windows, but we _can_ set assign it to `PROJECT_LOCATION`?

Reply:
Thanks, I forgot the 5th line